css/properties/baseline-source.json

```
@@ -0,0 +1,42 @@
+{
+  "css": {
+    "properties": {
+      "baseline-source": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/baseline-source",
+          "spec_url": "https://w3c.github.io/csswg-drafts/css-inline-3/#baseline-source",
+          "support": {
+            "chrome": {
+              "version_added": "11"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": false,
```

Per https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#setting-experimental

```suggestion
            "experimental": true,
```